packages/lesswrong/components/async/EditorForm.jsx

```diff
@@ -49,10 +49,10 @@ const styleMap = theme => ({
 class EditorForm extends Component {
   constructor(props) {
     super(props);
-    this.plugins = this.initializePlugins(props.isClient);
+    this.plugins = this.initializePlugins(props.isClient, props.commentEditor);
```

Comment:
Not super important, but not obvious we should be conceptualizing it was "comment editor" and "not comment editor". Simple Editor, maybe? (seems more applicable to all the places we're going to want to use this).

Reply:
I was thinking about that, but then our styles are indeed structured into `UI`, `Posts` and `Comments` categories, and given that this is primarily about styling, keeping that abstraction seemed sensible.